redash/monitor.py

```diff
@@ -31,7 +31,8 @@ def get_queues():
     scheduled_queue_names = db.session.query(DataSource.scheduled_queue_name).distinct()
     query = db.session.execute(union_all(queue_names, scheduled_queue_names))
 
-    return ['celery'] + [row[0] for row in query]
+    general_rq_queues = map(lambda q: 'rq:queue:' + q, ['default', 'periodic', 'schemas'])
```

Comment:
Why the `rq:queue:` prefix? 

Also, is there a way to ask `rq` for all the active queues?

Reply:
Oh, is it because that's the Redis key?

Reply:
Yupp

Reply:
> Also, is there a way to ask rq for all the active queues?

Yeah, I refactored it and now it reports all queues (on the way, it made sense to remove the `rq:queue:` prefix due to not poking directly in Redis anymore).